### PR TITLE
Language selection

### PR DESCRIPTION
**Pull request by dbrattli**
Add language selection to enable compilation to other languages like Python and PHP. Ref #2339 and #2447

### CODE REVIEW

File: src/fable-standalone/src/Main.fs [hunk 1]
```diff
@@ -265,7 +266,8 @@ let init () =
                 if x.StartsWith("--define:") || x.StartsWith("-d:")
                 then x.[(x.IndexOf(':') + 1)..] |> Some
                 else None) |> Array.toList
-            let options = Fable.CompilerOptionsHelper.Make(define=define, ?typedArrays=typedArrays, ?typescript=typescript)
+            let language = typescript |> Option.map (fun ts -> match ts with | true -> TypeScript | _ -> JavaScript) |> Option.defaultValue JavaScript
+            let options = Fable.CompilerOptionsHelper.Make(define=define, ?typedArrays=typedArrays, language=language)
```
Comment (ncave) [inline src/fable-standalone/src/Main.fs:270]: @dbrattli Is this missing the `?` on the language arg?

Comment (dbrattli) [reply src/fable-standalone/src/Main.fs:270]: No, don't think so. The `language` is not an option as I use `defaultValue`. Btw. Tried first to propagate language settings but problems with deps and where the Language union should live to be visible for all projects. Currently it's in `Fable.AST/Plugins.fs` which makes it hard to get it into e.g Worker.fsproj. So decided to keep the `typescript: bool` for now for fable-standalone.

Comment (ncave) [reply src/fable-standalone/src/Main.fs:270]: @dbrattli That's fine, I'll fix it after this gets merged. Thanks!

Comment (dbrattli) [reply src/fable-standalone/src/Main.fs:270]: @ncave Sounds great!

**Comment by alfonsogarciacaro** [comment]
Looks good, thanks a lot!

**Comment by thinkbeforecoding** [comment]
There is still some work for some specific constructs, and working out imports in a more php idiomatic way.. But globaly, it works and generate nice php code 😁

**Comment by sandeepc24** [comment]
Compilation to Dart language might be a good option, just saying :-)

**Pull request merged**